salt/modules/pkgin.py

```diff
@@ -681,7 +682,7 @@ def file_dict(*packages):
                 continue  # unexpected string
 
     ret = {'errors': errors, 'files': files}
-    for field in ret:
+    for field in list(ret):
```

Comment:
Using ``list()`` here resolves the ``RuntimeError: dictionary changed size during iteration
`` exception.